.buildkite/pipeline.yml

```diff
@@ -629,12 +629,11 @@ steps:
 
       - label: "Unit: matrix multiplication recursion example (CPU)"
         key: cpu_matrix_multiplication_recursion
-        command: "julia --color=yes --check-bounds=yes --project=test test/MatrixFields/matrix_multiplication_recursion.jl"
+        command: "julia --color=yes --check-bounds=yes --project=.buildkite test/MatrixFields/matrix_multiplication_recursion.jl"
```

Comment:
Ah, sorry, I must have missed this environment when merging

Reply:
No problem!